test/integration/default/inspec.yml

```diff
@@ -11,4 +11,3 @@ supports:
   - platform-name: ubuntu
   - platform-name: centos
   - platform-name: fedora
-  - platform-name: opensuse
```

Comment:
Is there any good reason to remove openSUSE platform?

Reply:
@baby-gnu Good catch, I did this work about a week or two ago, before you enabled `opensuse`.  I'll just fix it at my end and send that through again.

Reply:
@baby-gnu Fixed now.